package.json

```diff
@@ -52,6 +52,7 @@
     "web3": "1.0.0-beta.36"
   },
   "scripts": {
+    "update": "git submodule update --init --recursive && npm ci",
```

Comment:
It should also run `npm ci` for the submodules. In mosaic contracts, we have added submodule for utilitytoken contracts.

Reply:
Agree. Done.